src/elements/DoubleSlider/DoubleSlider.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="DoubleSlider" :style="cssVars">
+    <div id="left-progressBar"></div>
+    <div id="right-progressBar"></div>
+    <input id="faderLeft" type="range" v-model="faderLeftValue"
+      v-bind:min="sliderMin"
+      v-bind:max="sliderMax"
+      v-bind:step="sliderStep"
+      v-bind:aria-valuemin="sliderMin"
+      v-bind:aria-valuemax="sliderMax"
+      v-bind:aria-valuenow="faderLeftValue">
+    <input id="faderRight" type="range" v-model="faderRightValue"
+      v-bind:min="sliderMin"
+      v-bind:max="sliderMax"
+      v-bind:step="sliderStep"
+      v-bind:aria-valuemin="sliderMin"
+      v-bind:aria-valuemax="sliderMax"
+      v-bind:aria-valuenow="faderRightValue">
+  </div>
+</template>
+
+<script>
+  import { library } from '@fortawesome/fontawesome-svg-core'
+  import { faKeyboard } from '@fortawesome/free-solid-svg-icons'
+  import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+  import Colored from '@/mixins/colored'
+  import Indicating from '@/mixins/indicating'
+  import Joined from '@/mixins/joined'
+  import Rounded from '@/mixins/rounded'
+  import Scaled from '@/mixins/scaled'
+  import Simplified from '@/mixins/simplified'
+
+  import Invertible from '@/mixins/invertible'
+  import Unactionable from '@/mixins/unactionable'
+
+  library.add(faKeyboard)
+
+  /**
+   * ### The double range slider is a input field very much like the regular range slider.
+   * the big diffrence is a double range slider allows users
+   * to select both a min and max.
+   * exp. The range slider can be useful for allowing users to select
+   * a specific price range when browsing products.
+   */
+  export default {
+    name: 'DoubleSlider',
+    data: function () {
+      return {
+        faderLeftValue: 90,
+        faderRightValue: 100
+      }
+    },
+    components: {
+      FontAwesomeIcon
+    },
+    props: {
+      sliderMax: {
+        type: Number,
+        default: 100
+      },
+      sliderMin: {
+        type: Number,
+        default: 0
+      },
+      sliderStep: {
+        type: Number,
+        default: 1
+      },
+      nobColor: {
```

Comment:
Colors should be taken using the mixin `Colored`.

Reply:
Still need to wrap my head around how mixins work.

Reply:
It's very simple actually. Let's say a number of components take props `color` and `shade`. So instead of writing these props in every component, we write them once in a mixin (with defaults, validators etc.) and whichever component takes these props just imports and uses that mixin.

So components like `Heading`, `Button` have this in the component definition
```js
mixins: [
  ...
  Colored
]
``` 
This automatically adds the two props to the component, without having to repeat code over and over again.